src/inputs/selectNext/selectNext.tsx

```diff
@@ -409,6 +434,10 @@ const SelectNext = React.forwardRef(function SelectNext(
                     )}
                 >
                     {label}
+
+                    {required && showRequiredIndicator ? (
+                        <span className={classes.requiredIndicator}>*</span>
+                    ) : null}
```

Comment:
We shouldn't need to check `required` here (on Line 438) -- just checking `showRequiredIndicator` should be sufficient.

Reply:
Fixed: https://github.com/saddlebackdev/react-cm-ui/pull/462/commits/a9b97e03bd4f25c81b8a7f8c2013bc6feb50f15c#diff-a86b373e2035b10f8f8a9550a5f1f02cc22e5be10743d74e35732aaaa88295b7L438-R438
Resolving.